cdc/api/open.go

```diff
@@ -319,11 +319,16 @@ func (h *openAPI) PauseChangefeed(c *gin.Context) {
 		Type: model.AdminStop,
 	}
 
-	_ = h.capture.OperateOwnerUnderLock(func(owner *owner.Owner) error {
-		owner.EnqueueJob(job)
+	// Use buffered channel to prevernt blocking owner.
+	done := make(chan error, 1)
```

Comment:
May be it is better to extract these codes(line 322 to line 331) into a func like `sendJobToOwner(ctx context)`.